software/fluidd.yml

```
@@ -0,0 +1,13 @@
+name: Fluidd
+website_url: https://docs.fluidd.xyz/
+description: Fluidd is a lightweight & responsive user interface for Klipper, the 3D printer firmware.
+licenses:
+  - GPL-3.0
+platforms:
+  - TypeScript
```

```suggestion
  - Docker
  - Nodejs
```

> Values for platform are the main server-side requirements for the software

https://docs.fluidd.xyz/installation